.github/workflows/changelog_reminder.yml

```
@@ -0,0 +1,23 @@
+on:
+  pull_request:
+    types: [opened, ready_for_review]
+name: Changelog Reminder
+jobs:
+  remind:
+    name: Changelog Reminder
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Changelog Reminder
+      uses: peterjgrainger/action-changelog-reminder@v1.3.0
+      with:
+        changelog_regex: '.github/CHANGELOG.md'
+        customPrMessage: |
+          Beep! Boop! I see you're missing a changelog! 
```

How do we feel about the Beep Boop? I know I suggested it, but I'm worried its too on-the-nose/cutesy 😆 

Alternatively, we could go with a Robot emoji?
```suggestion
         Hello. You may have forgotten to update the changelog! 🤖 
```

---

Not the biggest fan of the emoji, but otherwise the change is good

---

Okay, lets go for a version without the emoji:
```suggestion
          Hello. You may have forgotten to update the changelog!
```